src/execution/__tests__/subscribe-test.ts

```diff
@@ -421,6 +421,28 @@ describe('Subscription Initialization Phase', () => {
     expect(() => subscribe({ schema })).to.throw('Must provide document.');
   });
 
+  it('throws an error if document is missing when using positional arguments', async () => {
```

Comment:
Did you forget to remove this one?
Or am I missing something?
Probably worth to also mark this test as "Deprecated:"

Reply:
Forgot to remove. And now done in #3634